src/log-pcap.c

```diff
@@ -1011,6 +1041,9 @@ static TmEcode PcapLogDataInit(ThreadVars *t, const void *initdata, void **data)
     if (unlikely(td == NULL))
         return TM_ECODE_FAILED;
 
+    td->counter_write = StatsRegisterCounter("pcap_log.write", t);
+    td->counter_filtered = StatsRegisterCounter("pcap_log.filtered", t);
```

Comment:
Thoughts on these stats names? They are really here more to very that the filter is being applied.

`.write`: Number of packets written
`.filtered`: Number of packets filtered *out*. I realize this is odd as in tcpdump speak, filtered is usually the number that matched the filter.

Reply:
Is there an argument/ reason for them not being something like:
`.bpf_written`
`.bpf_filtered_out` ?
I feel this would make them more self-explanatory.

Or maybe...
`.bpf_filter_write`
`.bpf_filtered_out`
(trying to think of something that correlates to the ymal config option)

Reply:
A `bpf` filter may not always be set, in fact its probably mostly not set.  In which case `bpf_filter_write` doesn't have much meaning as there is no bpf filter :)  And I think its probably a good metric to have the packets written even if no filter exists.

Reply:
There are more reasons we may not write a packet (depth, pass rules, etc) so I think the filtered counter should indeed include bpf in the name.
`.pkts_written`, `.pkts_filtered_bpf`. Could also drop the `ptks_`

Reply:
So maybe for now:
```
.written
.filtered_bpf
```